types/prosemirror-model/index.d.ts

```diff
@@ -1380,6 +1380,10 @@ export class Schema<N extends string = any, M extends string = any> {
      * bound.
      */
     markFromJSON(json: { [key: string]: any }): Mark<Schema<N, M>>;
+    /**
+     * Get schema's node type by name. This method uses `schema.nodes` internally
+     */
```

Comment:
This is an internal API member, and documenting internal APIs is out of scope for this repository. 

This is generally considered contrary to best practices, but if you feel you need the types, this can be accomplished using module augmentation. Check out the Typescript docs for examples.

Reply:
ok, thanks!